client/types.go

```diff
@@ -134,12 +134,13 @@ type RPCTransaction struct {
 }
 
 type RosettaTxReceipt struct {
-	Type           uint8  `json:"type,omitempty"`
+	Type           uint8 `json:"type,omitempty"`
 	GasPrice       *big.Int
 	GasUsed        *big.Int
 	TransactionFee *big.Int
 	Logs           []*EthTypes.Log
 	RawMessage     json.RawMessage
+	Status         uint64 `json:"status"`
```

Comment:
where do you set RosettaTxReceipt.Status?

Reply:
@xiaying-peng 
In `op-rosetta`, this function: https://github.com/mdehoog/op-rosetta/blob/fb98fe771618650538a742f4429ff1005a0cf43d/pkg/client/client.go#L81